azkaban-db/src/main/sql/create.project_flow_recommendations.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE project_flow_recommendations (
+  id                    INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  project_id            INT         NOT NULL,
+  flow_id               VARCHAR(128),
```

Comment:
Why flow_id can be null?

Reply:
Good point! Copied from our project_flows table. Let me mark mine as not null.